src/posthog-surveys.ts

```diff
@@ -27,6 +28,23 @@ export class PostHogSurveys {
         }
     }
 
+    getMatchingUrl(url?: string): boolean {
+        if (!url) return true
+
+        // If the url string starts and ends with / it is meant to be a regular expression
+        if (url.startsWith('/') && url.endsWith('/')) {
+            const regexPattern = url.slice(1, -1)
+            if (_isValidRegex(regexPattern)) {
+                return new RegExp(regexPattern).test(window.location.href)
+            }
+        }
+        // If the url string has a wildcard, convert to a regular expression
+        if (url.includes('*')) {
```

Comment:
I'd propose removing this magic, and let users use proper regexes if they need regexes.